mypy/modulefinder.py

```diff
@@ -493,8 +493,10 @@ def compute_search_paths(sources: List[BuildSource],
     egg_dirs, site_packages = get_site_packages_dirs(options.python_executable)
     for site_dir in site_packages:
         assert site_dir not in lib_path
-        if site_dir in mypypath:
+        if any(p.startswith(site_dir) for p in mypypath):
```

Comment:
If site_dir is `/foo/bar`, should we allow things like `/foo/bar2` in the mypy path? I.e., maybe we should require that there is a path separator after the site_dir.

Reply:
OK, makes sense.

Reply:
The new version disables `/foo/bar`, `/foo/bar/`, `/foo/bar/baz`, but allows `/foo/bar2`.